EscapeRoomReservation.cs

```diff
@@ -0,0 +1,244 @@
+                    Thread.Sleep(2000);
+                }
+        }
+    }
+
+
+
+        //Functie die alle escaperooms uitprint, en vervolgens jouw de keus geeft om er een te kiezen
+        public static EscapeRoom RoomPicker(){
+            var rooms = AJsonable.GetAll<EscapeRoom>("EscapeRooms");
+            while (true){
+                EscapeRoomController.ShowRooms();
+                int x = MenuOptions("Pick the number of the room you want to choose\n", rooms.Count-1);
+                Console.Clear();
+                Thread.Sleep(1000);
+                Console.WriteLine("You picked the following room:\n");
+                Console.WriteLine(rooms[x]);
+                Console.WriteLine("\n");
+                Thread.Sleep(1000);
+                Console.WriteLine("Is your choice correct?");
+                if (MenuOptions("\n\n1. Yes\n2.No", 2) == 1) {
+                    return rooms[x];
+                }
+        }
+    }
+
+
+
+        //Pakt een string als instructie. Returnt het menu item.
+        public static int MenuOptions(string text, int range, int minRange = 0) {
+            int x;
+            while (true) {
+                Console.WriteLine(text);
+                x = int.Parse(Console.ReadLine());
+                if (x > range || x < minRange) {
+                    Console.WriteLine("\nInvalid option. Try again\n");
+                    Thread.Sleep(1000);
+                } else {
+                    return x;
+                }
+            }
+        }
+        
+        //Pakt de hoeveelheid spelers
+        public static int NumberOfPlayers(int MaxPlayers){
+            int x;
+            x = MenuOptions("How many players will you bring along?\n", MaxPlayers+1);
+            return x;           
+        }
+
+        //Pakt de tijd van een gebruiker en convert dit tot minuten
+        public static int tijd(){
+            int x;
+            int y;
+            x = MenuOptions("Within two steps you will specify the hour and minute on which you'd like to reserve\n\nPlease specify at which hour you want (9:00-17:00)", 16, 9);
+            y = MenuOptions("Now please specify at which minute", 59, 0);
+            return TijdConverter(x, y);
+        }
+        //wordt gebruikt in de tijd functie om een aantal minuten te returnen
+        public static int TijdConverter(int h, int m){
+            return m + h*60;
+            }
+
+        //wordt gebruikt om tijd in minuten aan de gebruiker te presenteren.
+        public static string tijdAndersom(int tijd){
+            string tijdString = "";
+            int uren = tijd / 60;
+            int minuten = tijd % 60;
+            if (minuten < 10) {
+                tijdString = tijdString + uren + ":0" + minuten;
+            } else{
+                tijdString = tijdString + uren + ":" + minuten;
+            }
+            return tijdString;
+            }
+        }
+
+
+    //class voor één boeking
```

Comment:
Ik weet niet zeker of meerdere classes in een bestand netjes is wellicht ff na checken